waspc/src/Analyzer/TypeDefinitions.hs

```diff
@@ -13,33 +14,16 @@ module Analyzer.TypeDefinitions
     addEnumType,
```

Comment:
It was exported so `TypeCheckerTests` could make `TypeDefinitions` values.

Reply:
Ahhhh. Well hmmmm. That is not the best, our interfaces should not be affected by tests. 
I think we really need to not export internals of TypeDefinitinos here.
Why don't tests import internals of TypeDefinitions from TypeDefinitions.Types?
Btw, would it make sense to put Types and Class under Internal? Or are they used directly somewhere instead of through TypeDefinitions?

Reply:
Created a TODO for this.